frontend/src/app/organization/organization.component.ts

```diff
@@ -45,11 +45,11 @@ export class OrganizationComponent implements OnInit {
         })
       );
 
-    this.contactStatus$ = this.organizationService.contactStatus$.pipe(skip(1));
+    this.contactStatus$ = this.organizationService.contactStatus$;
   }
 
   onContact(organizationContact: OrganizationContactPayload) {
-    this.organization$.pipe(take(1))
+    this.organization$
```

Comment:
This change introduces a bug. Quite severe, in fact.

Since `organization$` observable never completes, you need to unsubscribe, otherwise you'll end up with leaking subscriptions. To illustrate the problem, send the contact form, press previous in your browser and then next in the browser. You are sending contact form again (I did not run this code, but that's what probably happens). The more time you send the contact form, the worse it will be.

Even better than unsubscribing, you can  `take(1)`, which makes the observable complete after taking just one element - which is exactly what you need here. Please don't modify this part here.

Reply:
Thanks for the catch, I wasn't aware of the observable completion thing. Still learning RxJS.

As a sidenote: organization component is over-engineered here in my opinion and we could handle this thing differently altogether, but that's not the point of this PR. :)